src/imports.jl

```diff
@@ -205,9 +207,5 @@ end
 
 import cohomCalg_jll
 
-import Nemo: is_cyclo_type
-import Nemo: is_maxreal_type
-import Nemo: ZZModRing  # FIXME: remove if/once Nemo exports this
-import Nemo: zzModRing  # FIXME: remove if/once Nemo exports this
-import Nemo: FpField  # FIXME: remove if/once Nemo exports this
-import Nemo: fpField  # FIXME: remove if/once Nemo exports this
+import Nemo: FpField, fpField  # FIXME: remove if/once Nemo exports this
+export FpField, fpField  # FIXME: remove if/once Nemo exports this
```

Comment:
```suggestion
import Nemo: FpField, fpField  # FIXME: remove once https://github.com/Nemocas/Nemo.jl/pull/1789 is available
export FpField, fpField  # FIXME: remove once https://github.com/Nemocas/Nemo.jl/pull/1789 is available
```
adding a PR link makes it easier in the future to check if the FIXME can be taken care of